notebooks/modelling/ACP_modelling.ipynb

```diff
@@ -0,0 +1,1065 @@
+      "│       └── [4.0K Sep  6 16:01]  ../../data/acp/esm/train/acp_train_esm1v_mean\n",
+      "    │   ├── [4.0K Sep 10 20:35]  ../../data/acp/prose/test/acp_test_dlm_avg\n",
+      "    │   ├── [4.0K Sep 10 20:37]  ../../data/acp/prose/test/acp_test_dlm_max\n",
+      "    │   ├── [4.0K Sep 10 20:37]  ../../data/acp/prose/test/acp_test_dlm_sum\n",
+      "    │   ├── [4.0K Sep 10 20:37]  ../../data/acp/prose/test/acp_test_mt_avg\n",
+      "    │   ├── [4.0K Sep 10 20:38]  ../../data/acp/prose/test/acp_test_mt_max\n",
+      "    │   └── [4.0K Sep 10 20:38]  ../../data/acp/prose/test/acp_test_mt_sum\n",
+      "        ├── [4.0K Sep  5 15:10]  ../../data/acp/prose/train/acp_train_dlm_avg\n",
+      "        ├── [4.0K Sep  5 15:18]  ../../data/acp/prose/train/acp_train_dlm_max\n",
+      "        ├── [4.0K Sep  5 15:28]  ../../data/acp/prose/train/acp_train_dlm_sum\n",
+      "        ├── [4.0K Sep  5 15:44]  ../../data/acp/prose/train/acp_train_mt_avg\n",
+      "        ├── [4.0K Sep  5 15:48]  ../../data/acp/prose/train/acp_train_mt_max\n",
+      "        └── [4.0K Sep  5 15:50]  ../../data/acp/prose/train/acp_train_mt_sum\n"
+     ]
+    }
+   ],
+   "source": [
+    "!tree -nhDL 3 ../../data/\"{task}\"/ -df | grep 'esm1\\|mt\\|dlm'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6cb35809-aa48-4063-b159-e95a2b3e0dfd",
+   "metadata": {},
+   "source": [
+    "<br>  \n",
+    "\n",
+    "To get paths for embedding folders and fasta files we will use the function `get_emb_folders()`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "ca536a8c-eef6-415a-b586-56b909890dd1",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-09-26T00:51:45.658074Z",
+     "iopub.status.busy": "2022-09-26T00:51:45.657178Z",
+     "iopub.status.idle": "2022-09-26T00:51:45.829631Z",
+     "shell.execute_reply": "2022-09-26T00:51:45.828440Z",
+     "shell.execute_reply.started": "2022-09-26T00:51:45.658038Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "pt_folders, fa_files = mu.get_emb_folders(task)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "68ac1c2a-14c4-4cfe-90f8-9912c8cc6414",
+   "metadata": {},
+   "source": [
+    "## Modelling Loop"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e22e4c4-d6b9-4abc-8305-9e330565a547",
+   "metadata": {},
+   "source": [
+    "The modelling loop includes the following steps:\n",
+    "\n",
+    "1. Loop through train and test embedding folders\n",
+    "2. Run the function `read_embeddings()` for train embeddings to get `X_train` and `y_train`\n",
+    "3. Run the function `read_embeddings()` for test embeddings to get `X_test` and `y_test`\n",
+    "4. Define and print the output header\n",
+    "5. Use the function `fit_tune_CV()`to to do the following:\n",
+    "   - use above defined `pipelines` and `hp_grids` dictionaries and `GridSearchCV()` to get models\n",
+    "   - save the models with `joblib`\n",
```

Comment:
I noticed you don't actually use joblib in this notebook to save the models, did you remove that line of code on purpose?